identity-appsupport/src/commonMain/kotlin/com/android/identity/appsupport/ui/Icon.kt

```diff
@@ -0,0 +1,93 @@
+package com.android.identity.appsupport.ui
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountBalance
```

Comment:
Are these .filled and .outlined imports needed? It looks like every time we reference one of these, it's a child of Icon.
(...and we never reference .filled)

Reply:
Yes, they're needed. The Icons.Default is the one referenced .filled (`Default` is an alias for `.Filled`).